Packs/CortexResponseAndRemediation/Playbooks/playbook-SSO_Authentication_With_Suspicious_Characteristics.yml

```diff
@@ -0,0 +1,1266 @@
+id: SSO Authentication With Suspicious Characteristics
+version: -1
+name: SSO Authentication With Suspicious Characteristics
+description: |-
+  **This playbook addresses the following alerts**:
+  - SSO authentication attempt with suspicious characteristics.
+  - Successful SSO authentication with suspicious characteristics.
+
+  **Playbook Stages**:
+
+  **Triage**:
+  - Collect initial information about the user and the SSO authentication event.
+  - Validate whether the authentication proxy is linked to iCloud Relay.
+
+  **Investigation**:
+  - **Check IOCs Reputation**:
+    - Analyze the reputation of IP addresses associated with the alert.
+  - **Search Related Alerts**:
+    - Look for alerts related to the same user within the system to identify suspicious activity trends.
+  - **Check If User Is Risky**:
+    - Retrieve the user's risk score and evaluate high-risk indicators for suspicious activities.
+  - **Check User Agent**:
+    - Identify suspicious user agents used during the authentication attempts.
+  - **Check Okta Logs**:
+    - Retrieve Okta authentication logs for failed login attempts and suspicious authentication activities within the last day.
+
+  **Containment**:
+  - **Automatic Actions**:
+    - Clear user sessions if any suspicious evidence is found during the investigation.
+  - **Analyst Review**:
+    - Provide an analyst with findings for review and determine the appropriate action:
+      - No action required.
+      - Suspend the user in Okta.
+    - If the analyst chooses to suspend the user, their active sessions are cleared in Okta.
+
+  **Requirements**:
+  For the best results, it's recommended to ensure these integrations are configured and working:
```

Comment:
On line 403, it says you should have MsGraph User.  Perhaps that should be on this list also?

Reply:
No since this playbook designed only for MS-Graph